cmd/wallet-js-sdk/test/fixtures/docker-compose.yml

```diff
@@ -11,14 +11,10 @@ services:
     image: ${HUB_ROUTER_IMAGE}:${HUB_ROUTER_IMAGE_TAG}
     environment:
       - HUB_ROUTER_HOST_URL=${EDGE_ROUTER_HOST}:${EDGE_ROUTER_API_PORT}
-      - HUB_ROUTER_TLS_CACERTS=/etc/tls/ec-cacert.pem
```

Comment:
Any reason to remove TLS config ?

Reply:
it is just unit tests, not needed, sometimes for dev mode testing, developer has to manually import them to continue testing.

TLS makes sense when you launch an app, doesn't make sense for tests.

Reply:
Ok, makes sense for manual tests. If it was server-server communication, we can just add it to [trusted certs](https://github.com/hyperledger/aries-framework-go/blob/main/test/bdd/fixtures/agent-rest/docker-compose.yml#L31).